src/datasources/cache/redis.cache.service.spec.ts

```diff
@@ -24,15 +25,31 @@ const mockLoggingService = {
   warn: jest.fn(),
 } as unknown as ILoggingService;
 
+const configurationService = {
+  getOrThrow: jest.fn(),
+} as unknown as IConfigurationService;
+const mockConfigurationService = jest.mocked(configurationService);
+
 describe('RedisCacheService', () => {
   let redisCacheService: RedisCacheService;
+  let defaultExpirationTimeInSeconds: number;
 
   beforeEach(async () => {
     clearAllMocks();
+    defaultExpirationTimeInSeconds = faker.number.int();
+    mockConfigurationService.getOrThrow.mockImplementation((key) => {
+      if (key === 'expirationTimeInSeconds.default') {
+        return defaultExpirationTimeInSeconds;
+      }
+      throw Error(`Unexpected key: ${key}`);
+    });
+
     redisCacheService = new RedisCacheService(
       redisClientTypeMock,
       mockLoggingService,
+      mockConfigurationService,
     );
+    jest.useRealTimers();
```

Comment:
We set the fake timers only in one test so I'd move this to that test for now.

Reply:
Agreed, changed in [b92584c](https://github.com/safe-global/safe-client-gateway/pull/970/commits/b92584cf5c50470b34305c8c9afffcc884c9fb21)